meson.build

```diff
@@ -0,0 +1,7 @@
+project('range-v3', 'cpp', version: '0.3.6', license: 'Boost, libc++, Stepanov and McJones "Elements of Programming", SGI STL')
```

Comment:
It's a bit hard to read it in a single long line, would you mind wrapping it?

Reply:
Do you mean like wrap into the variable ?

Reply:
```
project('range-v3', 'cpp', version: '0.3.6',
        license: 'Boost, libc++, Stepanov and McJones "Elements of Programming", SGI STL')
```